src/emqx_ct_helpers.erl

```diff
@@ -18,11 +18,95 @@
 
 -include_lib("common_test/include/ct.hrl").
 
--export([ensure_broker_started/0, ensure_broker_stopped/0]).
+-export([set_config/1,
+         run_setup_steps/1,
+         reload/2,
+         start_apps/1,
+         stop_apps/1,
+         ensure_broker_started/0, 
+         ensure_broker_stopped/0]).
```

Comment:
Delete `ensure_broker_started` `ensure_broker_stopped` functions and `emqx_ct_broker` module